entangled/commands/new.py

```diff
@@ -0,0 +1,149 @@
+from typing import Optional, Union
+
+import argh  # type: ignore
+import argparse
+import logging
+import traceback
+
+from copier import run_copy  # type: ignore
+from copier.errors import CopierError, UnsafeTemplateError  # type: ignore
+from pathlib import Path
+from rich.console import Console
+from rich.table import Table
+
+from ..config.templates import templates as AVAILABLE_TEMPLATES
+from ..config.templates import Template
+
+description = """Create a new entangled project from a template.
+
+You can choose either one of the official supported, built-in templates (see:
+entangled new -l), or any other template that copier supports, see:
+https://copier.readthedocs.io/en/stable/reference/vcs/#copier.vcs.get_repo"""
```

Comment:
This string is not used yet, it should end up in a future "long help" that does not exist yet.